estuary/api/v1.py

```diff
@@ -227,12 +235,17 @@ def _get_partial_stories(results, reverse=False):
 
     # Adding the artifact itself if its story is not available
     if not all_results:
+        base_instance = estuary.utils.story.BaseStoryManager()
+        wait_times, total_wait_time = base_instance.get_wait_times([item])
         rv = {'data': [item.serialized_all], 'meta': {}}
         rv['meta']['story_related_nodes_forward'] = [0]
         rv['meta']['story_related_nodes_backward'] = [0]
         rv['meta']['requested_node_index'] = 0
         rv['meta']['story_type'] = story_manager.__class__.__name__[:-12].lower()
-        rv['meta']['wait_times'] = [0]
+        rv['meta']['wait_times'] = wait_times
+        rv['meta']['total_wait_time'] = total_wait_time
+        rv['meta']['total_processing_time'] = base_instance.get_total_processing_time([item])
```

Comment:
`format_story_results` uses `try/excepts` when it calls these. It'd be nice to have that here too.